### PR TITLE
chore: adding contributors guides

### PR DESCRIPTION
**Pull request by toumorokoshi**
To help with onboarding and migration of more AEPs, adding a contributors guide.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 We'd love to accept your patches and contributions to this project.
 
+## Importing an AEP from google.aip.dev
+
+Currently, the project is focused on importing AIPs from
+https://google.aip.dev/. You can contribute! the process is:
```
Comment (rofrankel) [inline CONTRIBUTING.md:8]:
```suggestion
https://google.aip.dev/. You can contribute! The process is:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
 
 We'd love to accept your patches and contributions to this project.
 
+## Importing an AEP from google.aip.dev
+
+Currently, the project is focused on importing AIPs from
```
Comment (rofrankel) [inline CONTRIBUTING.md:7]:
```suggestion
Currently, the project is focused on adopting AIPs from
```

I think maybe "importing" implies that we aren't modifying the content at all?  But I'm not sure this word changes it all that much. WDYT?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+   1. Consider
+      [ones also with the label "good first issue"](https://github.com/aep-dev/aep.dev/issues?q=is%3Aopen+label%3A%22good+first+issue%22+label%3Aadoption).
+2. Find the original AIP, which is suffixed with the number in the issue. For
+   example https://google.aip.dev/126.
+3. Click on "view source" on the page, click on "raw", and copy the content.
+4. Create a directory for the aep, split the content in to an aep.md an
+   aep.yaml. [Example PR](https://github.com/aep-dev/aep.dev/pull/77/files).
+5. Modify content
+   1. remove any Google-isms (e.g. replace domains with example.com)
+   1. extend any proto guidance to include both OpenAPI and proto
```
Comment (rofrankel) [inline CONTRIBUTING.md:20]:
```suggestion
   1. Extend any proto-specific guidance to include both OpenAPI and proto.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,6 +2,29 @@
...
+      representations.
+
+Some tips:
+
+- Less is more. If there's a hard disagreement about particular guidance,
```
Comment (rofrankel) [inline CONTRIBUTING.md:25]: We won't get to disagreement from steps 1-5; I think you need a step 6 about creating a PR, right?  And to whom should the PR be assigned?

**Review by rofrankel — COMMENTED** [review]
Minor comments, generally LGTM.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,6 +2,29 @@
 
 We'd love to accept your patches and contributions to this project.
 
+## Importing an AEP from google.aip.dev
+
+Currently, the project is focused on importing AIPs from
+https://google.aip.dev/. You can contribute! the process is:
+
+1. [Select an aip to adopt from the open adoption issues](https://github.com/aep-dev/aep.dev/labels/adoption).
```
[the remark this answers is not in the document]
Comment (toumorokoshi) [reply CONTRIBUTING.md:10]: might as well be precise - I'll say AIP adopting into an AEP.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -2,6 +2,29 @@
...
+   1. Consider
+      [ones also with the label "good first issue"](https://github.com/aep-dev/aep.dev/issues?q=is%3Aopen+label%3A%22good+first+issue%22+label%3Aadoption).
+2. Find the original AIP, which is suffixed with the number in the issue. For
+   example https://google.aip.dev/126.
+3. Click on "view source" on the page, click on "raw", and copy the content.
```
[the remark this answers is not in the document]
Comment (toumorokoshi) [reply CONTRIBUTING.md:15]: do you have a suggestion on how to improve this? "raw" on the GH UI page will go to the markdown page.

File: CONTRIBUTING.md (hunk 4 above)
Comment (toumorokoshi) [reply CONTRIBUTING.md:25]: will add the final step! I think we should configure the reviewers in such a way that one doesn't have to specify reviewers. But I'll add you and I as an escalation path.

**Review by mkistler — APPROVED** [review]
Looks good! 👍

File: CONTRIBUTING.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (rofrankel) [reply CONTRIBUTING.md:19]:
```suggestion
3. Locate and copy the original Markdown source in https://github.com/aip-dev/google.aip.dev/blob/master/aip, e.g. https://github.com/aip-dev/google.aip.dev/blob/master/aip/general/0126.md
```

Link text would look nicer but might obscure the point. :/

There's another way to do it, with instructions about clicking the big blue GitHub button in the top right and navigating the repo, but why not link it directly?

**Review by rofrankel — APPROVED** [review]

File: CONTRIBUTING.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (toumorokoshi) [reply CONTRIBUTING.md:19]: the direct link doesn't really help in terms of discoverability. I am aware of the github button it doesn't link directly to the file so, although it's very straightforward, it doesn't direct someone there.

I guess it's fool-proof enough to just say "find the file on GitHub" - likely anyone dicovering this repository will be able to do that. Will update.

[the remark this answers is not in the document]
Comment (toumorokoshi) [reply CONTRIBUTING.md:19]: Updated with some links to the general directory and the GH repo directly.

**Pull request merged**